web-registry/src/pages/Admin/Admin.tsx

```diff
@@ -1,38 +1,53 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import { useAuth0 } from '@auth0/auth0-react';
+import { Link, Outlet, useNavigate } from 'react-router-dom';
+import { OAuthError, useAuth0 } from '@auth0/auth0-react';
+import { Box } from '@mui/material';
 
 import { AdminNav } from '../../components/organisms';
 import isAdmin from '../../lib/admin';
 
 function Admin(): JSX.Element {
-  const { user } = useAuth0();
+  const { user, error } = useAuth0();
+  const isUserAdmin = isAdmin(user);
+  const navigate = useNavigate();
+
+  const authError = error as OAuthError;
+  if (
+    authError &&
+    authError.error_description &&
+    authError.error_description.indexOf('email_not_verified:') > -1
+  ) {
+    const email: string = authError.error_description.split(':')[1];
+    navigate(`/verify-email?email=${email}`);
```

Comment:
it's not being actively used on prod atm but this doesn't seem to work anymore (ie using `navigate` over `history.push`)

Reply:
Thanks, will check.

Reply:
I revert that to the previous behavior.